config/haproxy_template.cfg

```diff
@@ -34,6 +36,24 @@ defaults
         errorfile 504 /etc/haproxy/errors/504.http
 
 
+frontend graceful_stop_check
```

Comment:
I couldn't think of a way to get rid of this extra frontend just for checking if shutdown is happening